frontend/src/metabase-types/api/action-form-settings.ts

```diff
@@ -0,0 +1,89 @@
+import type Field from "metabase-lib/metadata/Field";
```

Comment:
nit: We structure `api` types by endpoints clojure files. Is there a corresponding file on the backend, or is it a part of `actions`?

Reply:
Oh, I haven't realized that. There's none; we broke it down because `actions` became too big. Let me merge them together for now and we can figure out what to do next later